src/styles/utils/variables.scss

```
@@ -0,0 +1,4 @@
+$header-height: 60px;
+$card-width: 200px;
+$card-margin: 48px;
+$duration-transform: 300ms;
```

it is transition-duration
```suggestion
$transition-duration: 300ms;
```